src/components/booking/partial/steps-actions.tsx

```diff
@@ -31,6 +30,7 @@ export const steps: any = [
   "uploadBackIdImage",
   "pay",
   "saveData",
+  "saveDataOnValidation",
```

Comment:
Im not sure about here.
We are suppose to have 2 set of steps list
`stepsbooking`
`stepsReuploadIdentity` 
Here it seems you will break the booking page with an extra not required step